src/main/java/uk/co/ramp/covid/simulation/util/RNG.java

```diff
@@ -5,21 +5,21 @@
 /** Singleton for managing random number generation */
 public class RNG {
     private static RandomDataGenerator rng = null;
+    private static int currentSeed;
```

Comment:
This pull request is reverting some of the changes that I made related to running on the HPC system, and I don't understand why.  This change in RNG.java is one example.  It was made in the commit in this PR with comment "Addressed review comments".
Let's talk about this in our meeting later today.